app/config/featureswitch/FeatureSwitch.scala

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config.featureswitch
+
+import config.featureswitch.FeatureSwitch.prefix
+
+sealed trait FeatureSwitch {
+  val name: String
+  val displayText: String
+}
+
+object FeatureSwitch {
+  val prefix = "feature-switch"
+
+  val switches: Set[FeatureSwitch]  = Set(
+    PropertyDetails,
+    PropertyEops,
+    BusinessEops,
+    Payment,
+    Statements,
+    Estimates,
+    Bills,
+    ReportDeadlines,
+    AccountDetails,
+    CalcBreakdown,
+    CalcDataApi,
+    ObligationsPage
+  )
+
+  def apply(str: String): FeatureSwitch =
+    switches find (_.name == str) match {
+      case Some(switch) => switch
+      case None => throw new IllegalArgumentException("Invalid feature switch: " + str)
+    }
+
+  def get(str: String): Option[FeatureSwitch] = switches find (_.name == str)
+
+}
+
+case object PropertyDetails extends FeatureSwitch {
```

Comment:
As part of the story it mentions "When moving over the feature switch do not convert any dead code over."
This feature switch is not used anywhere in the code, so should be removed.